go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/mongodb/mongodb-atlas-cli/atlascli
 
 go 1.22
```

Comment:
checked out to the branch and got:
- do we need to do any setup to be able to update master?
```
❯ make deps
==> Installing go.mod dependencies...
go mod download
go: go.mod requires go >= 1.22 (running go 1.21.1; GOTOOLCHAIN=local)
make: *** [deps] Error 1
```

Reply:
see https://github.com/mongodb/mongodb-atlas-cli/pull/2779#discussion_r1530657881

you need to install/set go1.22 however you are managing your go versions, I have swapped from asdf to using the homebrew version (`asdf global golang system`) but you can keep using asdf if that works for you

Reply:
>, I still get the error,

I can't reproduce before doing that change can you confirm you have updated your env correctly, as in reload the shell env, via something like `source ~/.zshrc`, or restarting your term, and if you still get an error what's the output of `go env`?

Please consider that 
```
go 1.22.1
```

should be equivalent to 
```
go 1.22
toolchain go1.22.1
```

Reply:
I think if it's not reproducible I can workaround to fix my machine, so I'll leave my stamp.